esda/losh.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+import warnings
+from scipy import sparse
+from scipy import stats
+from sklearn.base import BaseEstimator
+import libpysal as lp
+
+
+class losh(BaseEstimator):
```

Comment:
This should be `LOSH` keeping with style elsewhere.

Reply:
Changed to `LOSH` in https://github.com/pysal/esda/pull/139/commits/3b47da838127a6e9677158ac9d11a2b321a8bef9